### PR TITLE
Russian and Ukrainian translations

### PR DESCRIPTION
**Pull request by LvMsterfild**
Updated Russian
Added Ukrainian
Added Ukrainian flag image
Added Ukrainian to languages.xml

### CODE REVIEW

File: lang/languages.xml [hunk 1]
```diff
@@ -1,50 +1,51 @@
...
+        <language display_name="Dansk" locale="da_DK" wallet-language="English" flag="/lang/flags/denmark.png" qs="none"/>
+        <language display_name="Česky" locale="cs_CZ" wallet_language="English" flag="/lang/flags/czech.png" qs="none"/>
+        <language display_name="Slovensky" locale="sk_SK" wallet_language="English" flag="/lang/flags/slovakia.png" qs="none"/>
+        <language display_name="Arabic" locale="ar_AR" wallet_language="English" flag="/lang/flags/egypt.png" qs="none"/>
+        <language display_name="Slovenski" locale="sl_SI" wallet_language="English" flag="/lang/flags/slovenia.png" qs="none"/>
+        <language display_name="Srpski" locale="rs_RS" wallet_language="English" flag="/lang/flags/srbija.png" qs="none"/>
+        <language display_name="Català" locale="cat_ES" wallet_language="Catalan" flag="/lang/flags/catalonia.png" qs="none"/>
+        <language display_name="Turkish" locale="tr_TR" wallet_language="Turkish" flag="/lang/flags/tr.png" qs="none"/>
+	<language display_name="Українська" locale="uk_UA" wallet_language="Українська" flag="/lang/flags/ukraine.png" qs="none"/>
```
Comment (erciccione) [inline lang/languages.xml]: Please change the wallet language from `"Українська"` to `"English"`

File: translations/monero-core_ru.ts [hunk 2]
```diff
@@ -1,2526 +1,2537 @@
...
-        <source>Description &lt;font size=&apos;2&apos;&gt;( Optional )&lt;/font&gt;</source>
-        <translation>Описание &lt;font size=&apos;2&apos;&gt;( Необязательно )&lt;/font&gt;</translation>
-    </message>
-    <message>
-        <location filename="../pages/Transfer.qml" line="318"/>
-        <source>Saved to local wallet history</source>
-        <translation>Сохранено в локальной истории кошелька.</translation>
-    </message>
-    <message>
-        <location filename="../p
```
Comment (erciccione) [inline translations/monero-core_ru.ts]: Please add `:` as in the source string

Comment (erciccione) [inline translations/monero-core_ru.ts]: Please add `:` as in the source string

Comment (erciccione) [inline translations/monero-core_ru.ts]: Please add `:` as in the source string

Comment (erciccione) [inline translations/monero-core_ru.ts]: Please add `:` as in the source string

Comment (erciccione) [inline translations/monero-core_ru.ts]: Please add `:` as in the source string

Comment (erciccione) [inline translations/monero-core_ru.ts]: Please add `:` as in the source string

File: translations/monero-core_ua.ts [hunk 3]
```diff
@@ -0,0 +1,2537 @@
...
+    </message>
+    <message>
+        <location filename="../pages/Settings.qml" line="498"/>
+        <source>Debug info</source>
+        <translation>Налагоджувальна інформація</translation>
+    </message>
+    <message>
+        <location filename="../pages/Settings.qml" line="512"/>
+        <source>GUI version: </source>
+        <translation>Версія GUI</translation>
```
Comment (erciccione) [inline translations/monero-core_ua.ts]: Missing `:`

File: translations/monero-core_ua.ts [hunk 4]
```diff
@@ -0,0 +1,2537 @@
...
+    </message>
+    <message>
+        <location filename="../pages/Settings.qml" line="517"/>
+        <source>Embedded Monero version: </source>
+        <translation>Вбудована версія Monero</translation>
```
Comment (erciccione) [inline translations/monero-core_ua.ts]: missing `:`

File: translations/monero-core_ua.ts [hunk 5]
```diff
@@ -0,0 +1,2537 @@
...
+    <message>
+        <location filename="../pages/Sign.qml" line="389"/>
+        <source>Paste in or select from &lt;a href=&apos;#&apos;&gt;Address book&lt;/a&gt;</source>
+        <translation>Вставте сюди чи виберіть із &lt;a href=&apos;#&apos;&gt;Адресної книги&lt;/a&gt;</translation>
+    </message>
+    <message>
+        <location filename="../pages/Sign.qml" line="160"/>
+        <location filename="../pages/Sign.qml" line="317"/>
+        <source>Or file:</source>
+        <translation>Чи файл</translation>
```
Comment (erciccione) [inline translations/monero-core_ua.ts]: missing `:`

File: translations/monero-core_ua.ts [hunk 6]
```diff
@@ -0,0 +1,2537 @@
...
+        <source>Fast (x5 fee)</source>
+        <translation>Швидка (x5 комісія)</translation>
+    </message>
+    <message>
+        <location filename="../pages/Transfer.qml" line="197"/>
+        <source>Fastest (x41.5 fee)</source>
+        <translation>Найшвидша (x41.5 комісія)</translation>
+    </message>
+    <message>
+        <location filename="../pages/Transfer.q
```
Comment (erciccione) [inline translations/monero-core_ua.ts]: missing `:`

Comment (erciccione) [inline translations/monero-core_ua.ts]: missing `:`

Comment (erciccione) [inline translations/monero-core_ua.ts]: missing `:`

**Comment by luigi1111** [comment]
@erciccione there's still an extra merge commit, but I'm not that OCD about it.

**Comment by erciccione** [comment]
@luigi1111 wasn't able to remove that, and it also looks empty. Maybe the "squash and merge" option here on github will do it

**Pull request merged**